contrib/signet/README.md

```diff
@@ -17,3 +17,64 @@ Syntax: `getcoins.py [-h|--help] [-c|--cmd=<bitcoin-cli path>] [-f|--faucet=<fau
 If using the default network, invoking the script with no arguments should be sufficient under normal
 circumstances, but if multiple people are behind the same IP address, the faucet will by default only
 accept one claim per day. See `--password` above.
+
+miner
+=====
+
+To mine the first block in your custom chain, you can run:
+
+  cd src/
+  CLI="./bitcoin-cli -conf=mysignet.conf"
+  MINER="..contrib/signet/miner"
```

Comment:
typo here, should be ../contrib!

Reply:
{
  "address": "1LfV1tSt3KNyHpFJnAzrqsLFdeD2EvU1MK",
  "total_received": 2000001157774,
  "total_sent": 1000000957003,
  "balance": 1000000200771,
  "unconfirmed_balance": 0,
  "final_balance": 1000000200771,
  "n_tx": 56,
  "unconfirmed_n_tx": 0,
  "final_n_tx": 56,
  "hasMore": true,
  "txs": [
    {
      "block_hash": "0000000000000000000da4a0dcefe6ccea3a90e951b050164dcaaa3383bc7e60",
      "block_height": 664638,
      "block_index": 1756,
      "hash": "a1a15c5f3e53c81d2a4e1de05c74c7faed74a8c8d7f2036ffe32200ff2f2d4b0",
      "addresses": ["13GEnxfijEf8SzwL1yEFVh5QFbXBYcgRFt"]